drivers/w1/w1_ds2482-800_channel.c

```diff
@@ -164,8 +164,8 @@ static const struct w1_driver_api ds2482_driver_api = {
 		.w1_config.slave_count = W1_INST_SLAVE_COUNT(inst),                                \
 		.parent = DEVICE_DT_GET(DT_INST_PARENT(inst)),                                     \
 		.i2c_spec = I2C_DT_SPEC_GET(DT_INST_PARENT(inst)),                                 \
-		.reg_channel = UTIL_CAT(CHSL_IO, DT_INST_REG_ADDR(inst)),                          \
-		.reg_channel_rb = UTIL_CAT(CHSL_RB_IO, DT_INST_REG_ADDR(inst)),                    \
+		.reg_channel = UTIL_CAT(CHSL_IO, DT_REG_ADDR_INTERNAL(DT_DRV_INST(inst))),         \
+		.reg_channel_rb = UTIL_CAT(CHSL_RB_IO, DT_REG_ADDR_INTERNAL(DT_DRV_INST(inst))),   \
```

Comment:
this feels wrong:

1. using `_INTERNAL` externally doesn't look great
2. Using DT_DRV_INST should be always avoided: add missing _INST macros instead

Reply:
Yeah the `INTERNAL` thing was more about hopes and dreams of that actually being internal, we know how that ended, changed to `RAW`, open to better suggestions, I suck bad with naming.

Added ther INST variants, current state was more like "can this even pass CI?".